### PR TITLE
Update auth.rst

### PR DESCRIPTION
**Pull request by AlexStallen**
added the line ,
            'authorize' => array('Controller'),  

or else the isAuthorized function will never be called, this will be very confusing for new people doing the tutorial

### CODE REVIEW

File: en/tutorials-and-examples/blog-auth-example/auth.rst [hunk 1]
```diff
@@ -354,7 +354,8 @@ config::
                 'controller' => 'Pages',
                 'action' => 'display',
                 'home'
-            ]
+            ],
+            'authorize' => array('Controller'),
```
Comment (dereuromark) [inline en/tutorials-and-examples/blog-auth-example/auth.rst]: Short array syntax: [...]

Comment (AlexStallen) [inline en/tutorials-and-examples/blog-auth-example/auth.rst]: yeah sorry, still used to php 5.3...

File: en/tutorials-and-examples/blog-auth-example/auth.rst [hunk 2]
```diff
@@ -354,7 +354,8 @@ config::
                 'controller' => 'Pages',
                 'action' => 'display',
                 'home'
-            ]
+            ],
+            'authorize' => ['Controller'],
```
Comment (markstory) [inline en/tutorials-and-examples/blog-auth-example/auth.rst]: This is already present on line 348.

**Comment by markstory** [comment]
The line you want to add is already in that code snippet with a comment indicating it was added.

**Pull request closed without merging**